src/Navigation/Dropdown/Dropdown.tsx

```diff
@@ -205,17 +202,33 @@ type PropsFromDropdownBody = Omit<
   'open'
 >;
 
-interface Props
+export interface Props
   extends PropsFromDropdownContainer,
     PropsFromDropdownHeader,
     PropsFromDropdownBody {
   children: React.ReactNode;
+  /** Sets label of Dropdown. */
   label: string;
+  /** Opens Dropdown on hovering. */
   hoverToOpen?: boolean;
-  leftIcon?: React.ReactType;
+  /** Icon to show at the left side of dropdown button. */
```

Comment:
Had to change the type, else in the table the examples and the type wasn't proper.